apps/package.json

```diff
@@ -11,14 +11,14 @@
     "lint": "eslint . --ext .js,.jsx",
     "test": "npm run lint && grunt test",
     "test-low-memory": "./test-low-memory.sh",
-    "test:unit": "grunt unitTest",
-    "test:integration": "grunt integrationTest",
+    "test:unit": "grunt unitTest; grunt logBuildTimes",
+    "test:integration": "grunt integrationTest; grunt logBuildTimes",
     "clean": "grunt clean",
-    "build": "MOOC_DEV=1 grunt build",
-    "build:dist": "MOOC_DEV=0 grunt clean build",
-    "start": "MOOC_DEV=1 grunt dev",
-    "start:applab": "npm run start -- --app=applab",
-    "start:gamelab": "npm run start -- --app=gamelab",
+    "build": "MOOC_DEV=1 grunt build; grunt logBuildTimes",
+    "build:dist": "MOOC_DEV=0 grunt clean build; grunt logBuildTimes",
+    "start": "MOOC_DEV=1 grunt dev; grunt logBuildTimes",
+    "start:applab": "MOOC_DEV=1 grunt dev --app=applab; grunt logBuildTimes",
+    "start:gamelab": "MOOC_DEV=1 grunt dev --app=gamelab; grunt logBuildTimes",
```

Comment:
So, running `npm run start` and hitting <kbd>Ctrl</kbd>+<kbd>C</kbd> no upload appears to occur - I'm not sure if NPM is swallowing the log output (unlikely) or otherwise interrupting the task (more likely?) but not sure it works.

If, however, I run `MOOC_DEV=1 grunt dev; grunt logBuildTimes` directly and hit <kbd>Ctrl</kbd>+<kbd>C</kbd> this works as advertised.

Worth trying: Set up `logBuildTimes` as a `postbuild`, `poststart` etc step instead.  That may restore your ability to chain the start tasks somewhat, too.